src/components/AndelaCentres/AndelaCentresComponent.jsx

```diff
@@ -41,45 +42,57 @@ class AndelaCentresComponent extends React.Component {
 
     return (
       <NavBarComponent>
-        <PageHeader header="Andela Centres" />
-
         <div className="assets-list">
-          {showStatus && (
-            <StatusMessageComponent
-              message={error}
-              className="error-status"
-              reset={resetMessage}
+          <Header as="h1" id="page-headings" floated="left" content="Andela Centres" />
+          <Divider id="assets-divider" />
+          <div className="header-modal-button">
+            <ModalComponent
+              trigger={
+                <Button className="add-asset" size="medium">
```

Comment:
Also out of curiosity, all the trigger buttons that are used to show the modals have specific class names such as `add-asset`. Can't you make the class name more generic? I believe that all the trigger buttons are of similar size, therefore generalizing the class name should not make much of a difference